plugins/generator-datapack-1.18.x/datapack-1.18.x/templates/recipes/blasting.json.ftl

```diff
@@ -0,0 +1,13 @@
+<#-- @formatter:off -->
+<#include "../mcitems.ftl">
+{
+    <#if data.group?has_content>"group": "${data.group}",</#if>
+    "type": "minecraft:blasting",
+    "experience": ${data.xpReward},
+	"cookingtime": ${data.cookingTime},
+    "ingredient": {
+      ${mappedMCItemToIngameItemName(data.blastingInputStack)}
+    },
+    "result": "${mappedMCItemToIngameNameNoTags(data.blastingReturnStack)}"

```

Comment:
mappedMCItemToIngameNameNoTags already contains ", was this PR tested as these recipes will not work?